Userland/Utilities/ping.cpp

```diff
@@ -180,6 +183,14 @@ int main(int argc, char** argv)
                 return 1;
             }
 
+            char internet_header_length = -1;
+            internet_header_length = *pong_packet.data() & 0x0F;
+            if (internet_header_length < min_header_size_in_bytes) {
+                outln("ihl illegal {0:x}", internet_header_length);
```

Comment:
this doesn't seem like a very nice message for the user, something like `warnln("ping: illegal ihl field value {:x}", internet_header_length);` would be nicer to see.

(also, no need to explicitly set the field index in AK/Format, `{:x}` is sufficient)